pandas/tests/io/json/test_pandas.py

```diff
@@ -1242,6 +1243,21 @@ def test_read_jsonl_unicode_chars(self):
         expected = DataFrame([["foo\u201d", "bar"], ["foo", "bar"]], columns=["a", "b"])
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize("bigNum", [sys.maxsize + 1, -(sys.maxsize + 2)])
+    def test_to_json_large_numbers(self, bigNum):
+        # GH34473
+        originalSeries = Series(bigNum, dtype=object, index=["articleId"])
```

Comment:
can you use series here (and just df below)

Reply:
sure!